symplyphysics/laws/kinematic/angular_position_is_arc_length_over_radius.py

```diff
@@ -27,6 +30,38 @@
 law = Eq(angular_position, arc_length / path_radius)
 
 
+# Derive law from its [vector counterpart](../linear_displacement_is_angular_displacement_cross_radius.py)
+
+angle_x = Symbol("angle_x", angle_type)
+angle_y = Symbol("angle_y", angle_type)
+angle_z = Symbol("angle_z", angle_type)
+angle_vec = Vector([angle_x, angle_y, angle_z])
+angle_norm = vector_magnitude(angle_vec)
+
+radius_x = Symbol("radius_x", units.length)
+radius_y = Symbol("radius_y", units.length)
+radius_z = Symbol("radius_z", units.length)
+radius_vec = Vector([radius_x, radius_y, radius_z])
+radius_norm = vector_magnitude(radius_vec)
+
+displacement_from_law = solve(
+    law, arc_length
+)[0].subs({
+    angular_position: angle_norm,
+    path_radius: radius_norm,
+})
+
+# From [the law](../linear_displacement_is_angular_displacement_cross_radius.py), taking the vector
+# norm of both sides, we have norm(s) = norm(theta) * norm(r) * sin(phi) where s is the linear
+# displacement vector, theta is the angular displacement pseudovector, r is the radius vector, and
+# phi is the and between the last two. As a necessary condition, theta and r are orthogonal to each
+# other, therefore phi is pi/2 and sin(phi) is 1, and so abs(s) = abs(theta) * abs(r)
+
+displacement_derived = angle_norm * radius_norm
```

Comment:
You should derive it from the law, not hardcode it.

Reply:
Well, that was a sly hack of mine, I agree. I will try and add laws for cross and dot products since they are often needed but there is nothing to refer to in code.

Reply:
But you're referring cross product law. Why not use it for proof?